src/zcl_abapgit_transport_mass.clas.abap

```diff
@@ -0,0 +1,262 @@
+CLASS zcl_abapgit_transport_mass DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    CONSTANTS gc_fm_name TYPE rs38l-name VALUE 'Z_ABAPGIT_TRANSPORTS_2_ZIP' ##NO_TEXT.
+
+    CLASS-METHODS run .
+    CLASS-METHODS zip
+      IMPORTING
+        !is_trkorr     TYPE trwbo_request_header
+        !iv_logic      TYPE string OPTIONAL
+      RETURNING
+        VALUE(rv_xstr) TYPE xstring
+      RAISING
+        zcx_abapgit_exception .
+    CLASS-METHODS to_tadir
```

Comment:
looks like some of these methods are copied from https://github.com/larshp/abapGit/blob/master/src/zcl_abapgit_transport.clas.abap#L22 ? please refactor instead of duplicating

Reply:
Hi,
         I refactored the code to avoid redundancy.
Regards,
Thomas.